examples/read_external_data.py

```
@@ -0,0 +1,75 @@
+        print(f"   Date range: {openaq_data['datetime'].min()} to {openaq_data['datetime'].max()}")
+    except FileNotFoundError as e:
+        print(f"   Error: {e}")
+    
+    print()
+    
+    try:
+        print("2. Reading Japan weather data (Jan 2023)...")
+        weather_data = reader.read_weather(
+            source='openmeteo',
+            country='JP',
+            start_date=datetime(2023, 1, 1),
```

Ruff (0.12.2)

`datetime.datetime()` called without a `tzinfo` argument

(DTZ001)